.github/workflows/ci.yml

```diff
@@ -286,6 +286,7 @@ jobs:
           ./build_tools/github_actions/docker_run.sh \
             --env IREE_VULKAN_F16_DISABLE=0 \
             --env IREE_CUDA_DISABLE=0 \
+            --env IREE_NVIDIA_GPU_TESTS_DISABLE=0 \
```

Comment:
Should maybe also enable in `test_tf_integrations_gpu` + [`run_tf_tests.sh`](https://github.com/openxla/iree/blob/main/build_tools/cmake/run_tf_tests.sh) somehow? Could do the plumbing now, so that if affected tests are added they get covered instead of silently ignored.

Reply:
Ah, update both the ci.yml and the tf script
(looks like your latest commit just updates the script)

Reply:
I added it to `test_tf_integration_gpu`, but I didn't see any tests with `requires-nvidia-gpu` in `integrations/tensorflow/test`, so I'm not sure what else there is to change?

Reply:
Yep that looks correct, thanks! No tests at the moment, but if we add some we'd want them to start working automatically.